frontend/src/language/en.json

```diff
@@ -0,0 +1,65 @@
+{
+    "Ongoing missions": "Ongoing missions",
+    "Upcoming missions": "Upcoming missions",
+    "Past Missions": "Past Missions",
+    "failed on robot": "failed on robot",
+    "missions failed recently. See 'Past Missions' for more information.": "missions failed recently. See 'Past Missions' for more information.",
+    "Task": "Task",
+    "No ongoing missions": "No ongoing missions",
+    "No pending missions in queue": "No pending missions in queue",
+    "Status": "Status",
+    "Name": "Name",
+    "Completion Time": "Completion Time",
+    "Please select asset": "Please select asset",
+    "This asset has no missions - Please create mission": "This asset has no missions - Please create mission",
+    "Cancel": "Cancel",
+    "Schedule mission": "Schedule mission",
+    "Schedule Missions": "Schedule Missions",
```

Comment:
Should stay consequent whether it is Schedule missions or Schedule mission.
I propose missions in both, as it is possible to schedule multiple in one go.

Reply:
Changing it to "Select missions" as the following lines are "Select robot" and "Select start time"